CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+## 0.9.0-alpha+3
+
+> NOTE: This code is considered production quality, but depends on angular:
+> 5.0.0-alpha+3. The alpha tag represents the evolving nature of the AngularDart
+> api, not code quality (5.0.0-alpha+3 is used in production Google apps).
```

Comment:
The Angular version was bumped up (from +3 to +4) in the pubspec.yaml, just before merging. This changelog entry should be referring to 5.0.0-alpha+4.